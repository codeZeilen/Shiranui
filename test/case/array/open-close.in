let a = [1..10);
let b = [1,2..10);
let c = [10,9..1);
let d = [10..1);

let sum = \(ar){
    mut ret = 0;
    for i in ar {
        ret <- !ret + i;
    }
    return !ret;
};

let unit = print(sum(a));
let unit = print(sum(b));
let unit = print(sum(c));
let unit = print(sum(d));
