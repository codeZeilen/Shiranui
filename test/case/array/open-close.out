45
45
54
0
